### DISCUSSION TITLE
Remember column reorder per table

### DISCUSSION THREAD

**Discussion by bombillazo** [dbeaver/dbeaver#35662] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**
Some DBs (like PostgreSQL) do not allow for column position changing in a table. This makes it hard to organize columns since their order will depend on how the table was created.

**Describe the solution you'd like**
It would be great for DBeaver to render columns and store the order as the users change it in the GUI so that the order is maintained the next time the user opens the table.

**Describe alternatives you've considered**
Having to create a view, which is more work than DBeaver simply remembering the order and introduces a new DB entity simply for presentation purposes

**Comment by E1izabeth** [comment, marked as answer]
This feature exists. Here is a short video showing how to use it:


https://github.com/user-attachments/assets/165aa0f2-ff83-44f3-931c-17e62a4b8965

**Comment by bombillazo** [comment]
Oh excellent! Did not know about this at all, this is a game changer for me, thanks!

**Comment by dvoracek-jakub** [comment]
This works, until you click the "Remove all filters/orderings" button just next to "Customize filters" button. Then your custom ordering is lost and you can reorder it manually again.

**Comment by haldi4803** [comment]
Oh wow.... so it's not a "saved layout" or stuff but a simple Order in the Filter Settings!
Thanks a lot.
